lib/internal/fs/promises.js

```diff
@@ -8,6 +8,9 @@ const kReadFileBufferLength = 512 * 1024;
 const kReadFileUnknownBufferLength = 64 * 1024;
 const kWriteFileMaxChunkSize = 512 * 1024;
 
+// 2 ** 32 - 1
+const kMaxUserId = 4294967295;
```

Comment:
Why this value? Is this documented somewhere? Could you add a comment explaining where does it come from please?

Reply:
I actually pasted it from: https://github.com/nodejs/node/blob/82bc5c3d5c8645d01fe26ade1f75d926a6fb40ae/lib/fs.js#L139-L140
since I could not require it because of a circular dependency.
Should we move this (and possibly other k* constants) into `lib/internal/fs/utils.js` and require them from there?